applications/app/controllers/OptInController.scala

```diff
@@ -31,10 +31,12 @@ class OptInController extends Controller {
     Cached(60)(WithoutRevalidationResult(feature match {
       case "headerseven" => headerSeven.opt(choice)
       case "youtubeposter" => youtubePosterOverride.opt(choice)
+      case "abnewrecipedesign" => ABNewRecipeDesignOverride.opt(choice)
```

Comment:
Nit: this doesn't have the AB prefix and doesn't need it. I probably would remove it for consistency sake